src/styles/colors.scss

```diff
@@ -22,6 +22,7 @@ $scTeal:            #56c4c4;
 $scTealDark:       #329594;
 $scGreen:           #54cd86;
 $scGreenDark:      #398f5d;
+$scGreenOffDark:    #3a8e5d;
```

Comment:
The most nitpickiest nitpick ever ... could we align the color hex values? ;-)
Feel free to me tell me "Geoffrey, with all due respect, no dude!  I'm not gonna do that!"